test/Api.Test/Vault/AuthorizationHandlers/BulkCollectionAuthorizationHandlerTests.cs

```diff
@@ -134,7 +144,6 @@ public class BulkCollectionAuthorizationHandlerTests
         CurrentContextOrganization organization)
     {
         organization.Type = userType;
-        organization.LimitCollectionCreationDeletion = true;
```

Comment:
This line (whether true or false) has been removed from a lot of tests with no replacement. Is that intentional? In some cases, I think it's required to make sure that the test is testing what we want it test. (test test test) They're not failing, but we also need to think about false negatives.

I suggest adding a private method that tests can call to arrange this value.

Reply:
This is intentional. However, I went back through all of these tests revolving around needing the `LimitCollectionCreationDeletion` property and added some explicit calls to make sure we never get flaky tests.

Reply:
left a further comment about this